File: src/luogu/p5706/Main.cpp
// https://www.luogu.com.cn/problem/P5706

#include <cstdio>

int main() {
  float a;
  int b;
  scanf("%f %i", &a, &b);
  printf("%.3f\n%i\n", a / b, b * 2);
  return 0;
}

File: src/luogu/p5706/Main.java
// https://www.luogu.com.cn/problem/P5706
package luogu.p5706;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double t = sc.nextDouble();
        int n = sc.nextInt();
        double p = t / n;
        System.out.printf("%.3f\n", p);
        System.out.println(2 * n);
    }
}
